src/v2/Apps/Order/redirects.tsx

```diff
@@ -278,5 +264,6 @@ graphql`
     creditCard {
       internalID
     }
+    ...Payment_validation @relay(mask: false)
```

Comment:
Question: Do we still need `mask: false` in both places if the components use them directly? Or maybe I missed something here.

Reply:
If I'm not mistaken then yes, we have to use `mask: false` to get access to the fields described in the fragment from the generated field. In another case, we will have only link to the fragment and we don't have access to the fields.